### PR TITLE
Add InSpec Builder

### PR DESCRIPTION
**Pull request by henryken**
Add InSpec builder.

### CODE REVIEW

**Comment by henryken** [comment]
@nof20, would you be able to help review this?

File: inspec/examples/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+```bash
+export PROJECT=$(gcloud info --format='value(config.project)')
+export PROJECT_ID=$(gcloud projects describe $PROJECT --format 'value(projectId)')
+export MEMBER=$(gcloud projects describe $PROJECT --format 'value(projectNumber)')@cloudbuild.gserviceaccount.com
+gcloud projects add-iam-policy-binding $PROJECT --member=serviceAccount:$MEMBER --role='roles/compute.networkViewer'
+```
+
+Update `test-profile/attributes.yml` to point to your GCP project
+```bash
+sed -i .ori 's/my-gcp-project/'"$PROJECT_ID"'/g' test-profile/attributes.yml
```
Comment (nof20) [inline inspec/examples/README.md]: Think you have a typo in here - command should be:

```
sed -i 's/my-gcp-project/'"$PROJECT_ID"'/g' test-profile/attributes.yml 
```

Comment (henryken) [reply inspec/examples/README.md]: Ah yeah it's Mac specific flag. I just removed it. Thanks for pointing out.

**Comment by nof20** [comment]
Looks good.  Thanks!

**Pull request merged**